dist/index.html

```diff
@@ -2,11 +2,197 @@
+          </div>
+          <div class="filter">
+            <input type="radio" id="breakfast" name="tags" value="breakfast">
+            <label for="breakfast">Breakfast</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="brunch" name="tags" value="brunch">
+            <label for="brunch">Brunch</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="condiment" name="tags" value="condiment">
+            <label for="condiment">Condiment</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="dinner" name="tags" value="dinner">
+            <label for="dinner">Dinner</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="dip" name="tags" value="dip">
+            <label for="dip">Dip</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="hor-d'oeuvre" name="tags" value="hor-d'oeuvre">
+            <label for="hor-d'oeuvre">Hor d'oeuvre</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="lunch" name="tags" value="lunch">
+            <label for="lunch">Lunch</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="main-course" name="tags" value="main-course">
+            <label for="main-course">Main Course</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="main-dish" name="tags" value="main-dish">
+            <label for="main-dish">Main Dish</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="morning-meal" name="tags" value="morning-meal">
+            <label for="morning-meal">Morning Meal</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="salad" name="tags" value="salad">
+            <label for="salad">Salad</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="sauce" name="tags" value="sauce">
+            <label for="sauce">Sauce</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="side-dish" name="tags" value="side-dish">
+            <label for="side-dish">Side Dish</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="snack" name="tags" value="snack">
+            <label for="snack">Snack</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="spread" name="tags" value="spread">
+            <label for="spread">Spread</label>
+          </div>
+          <div class="filter">
+            <input type="radio" id="starter" name="tags" value="starter">
+            <label for="starter">Starter</label>
+          </div>
+        </form>
+      </section>
+      <section class="ingredient-section">
+        <h2 class="ingredient-title">Ingredients</h2>
```

Comment:
I don't know this for sure, but I feel like this will also be an innerHTML problem